src/ng/directive/ngModelOptions.js

```diff
@@ -331,19 +331,28 @@ defaultModelOptions = new ModelOptions({
  *
  */
 var ngModelOptionsDirective = function() {
+  NgModelOptionsController.$inject = ['$attrs', '$scope'];
+  function NgModelOptionsController($attrs, $scope) {
+    this.$$attrs = $attrs;
+    this.$$scope = $scope;
+ }
+  NgModelOptionsController.prototype = {
+    $onInit: function() {
+      var parentCtrl = this.parentOptionsCtrl;
```

Comment:
given that we can name the binding anything we like why not just call it `parentCtrl` and then there would be very little motivation for this line any more.